data/aliases.js

```diff
@@ -236,20 +237,31 @@ exports.BattleAliases = {
 
 	// pokemon
 	"aboma": "Abomasnow",
+	"alaka": "Alakazam",
+	"ape": "Infernape",
 	"bliss": "Blissey",
 	"chomp": "Garchomp",
 	"cofag": "Cofagrigus",
+	"conk": "Conkeldurr",
+	"croak": "Toxicroak",
 	"cress": "Cresselia",
 	"dnite": "Dragonite",
 	"don": "Groudon",
 	"dogars": "Koffing",
+	"dos": "Gyarados",
 	"ekiller": "Arceus",
 	"esca": "Escavalier",
 	"ferro": "Ferrothorn",
+	"ferry": "Ferrothorn",
 	"forry": "Forretress",
 	"gar": "Gengar",
 	"garde": "Gardevoir",
+	"gross": "Metagross",
+	"gyara": "Gyarados",
+	"hax": "Haxorus",
```

Comment:
"hax" is an actual term in Pokemon, so I'm against it.

`haxor` imo lol, and maybe `haxo` but that one's stretching it